src/modules/75-cd/components/EnvironmentGroups/EnvironmentGroupsListColumns.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React from 'react'
+import ReactTimeago from 'react-timeago'
+import { useParams } from 'react-router-dom'
+import { defaultTo, isEmpty } from 'lodash-es'
+import { Classes, Intent, Menu, Position } from '@blueprintjs/core'
+
+import { Button, Layout, Popover, TagsPopover, Text, useConfirmationDialog } from '@harness/uicore'
+import { Color } from '@harness/design-system'
+import { useStrings } from 'framework/strings'
+
+import RbacMenuItem from '@rbac/components/MenuItem/MenuItem'
+import { usePermission } from '@rbac/hooks/usePermission'
+import { ResourceType } from '@rbac/interfaces/ResourceType'
+import { PermissionIdentifier } from '@rbac/interfaces/PermissionIdentifier'
+
+import { EnvironmentType } from '@common/constants/EnvironmentType'
+
+import css from './EnvironmentGroups.module.scss'
+import environmentCss from '../Environments/EnvironmentsListColumns/EnvironmentsListColumns.module.scss'
+
+function EnvironmentGroupName({
+  name,
+  identifier,
+  tags
+}: {
+  name?: string
+  identifier?: string
+  tags?: {
+    [key: string]: string
+  }
+}): React.ReactElement {
+  const { getString } = useStrings()
+
+  return (
+    <>
+      <Layout.Horizontal flex={{ justifyContent: 'flex-start' }} spacing="small" margin={{ bottom: 'small' }}>
+        <Text color={Color.BLACK} lineClamp={1}>
+          {name}
+        </Text>
+        {!isEmpty(tags) && (
+          <TagsPopover
+            className={css.tagsPopover}
+            iconProps={{ size: 14, color: Color.GREY_600 }}
+            tags={defaultTo(tags, {})}
+            popoverProps={{
+              position: 'right'
+            }}
+          />
+        )}
+      </Layout.Horizontal>
+
+      <Text color={Color.GREY_500} font={{ size: 'small' }} lineClamp={1}>
+        {getString('common.ID')}: {identifier}
+      </Text>
+    </>
+  )
+}
+
+function LastUpdatedBy({ lastModifiedAt }: { lastModifiedAt?: number }): React.ReactElement {
+  return <ReactTimeago date={lastModifiedAt as number} />
+}
+
+function NoOfEnvironments({ length }: { length: number }) {
+  return (
+    <Text
+      color={Color.BLACK}
+      background={Color.GREY_100}
+      padding={{ left: 'large', top: 'small', right: 'large', bottom: 'small' }}
+      inline
+    >
+      {length} environments included
+    </Text>
+  )
+}
+
+function EditOrDeleteCell({ identifier, onEdit, onDelete }: any): React.ReactElement {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232281)
```suggestion
function EditOrDeleteCell({ identifier, onEdit, onDelete }: unknown): React.ReactElement {
```